project-koku/koku
# Cost 2389 apis openapi

Fixes [COST-2389](https://issues.redhat.com/browse/COST-2389)

Changes proposed in this PR:
* Adding API's for OCI
* Updating Openapi spec for OCI API's

Testing Instructions:
1. Add OCI provider/ingest some data
2. Navigate to new OCI endpoints and confirm they load correctly
   ```
   GET http://localhost:8000/api/cost-management/v1/reports/oci/costs/

   {json: response}
   ```
3. Confirm openapi spec is updated accordingly

---
Additional Context:

Please reach out for adding OCI provider
